src/Umbraco.PublishedCache.HybridCache/Services/MediaCacheService.cs

```diff
@@ -166,7 +167,7 @@ public async Task SeedAsync(CancellationToken cancellationToken)
 
             var cacheKey = GetCacheKey(key, false);
 
-            ContentCacheNode? cachedValue = await _hybridCache.GetOrCreateAsync<ContentCacheNode?>(
+            ContentCacheNode? cachedValue = await _hybridCache.GetOrCreateAsync(
```

Comment:
There's a comment above for "Always set draft node" which I don't see is relevant for media. It's not really part of this PR, but could we remove/adjust?

Reply:
Removed 🆗